frontend/test/resolution/testIf.cpp

```diff
@@ -0,0 +1,284 @@
+/*
+ * Copyright 2021-2022 Hewlett Packard Enterprise Development LP
```

Comment:
Remember to update the copyright year of the new files.  :joy:

Reply:
Automatically taken care of with the rebase 😄.

Reply:
Hmm.  I don't know how it could have done that.  And it hasn't done it to test-common.cpp yet.

Reply:
Not only that - but I can't seem to get CI checks to fire...